packages/collector/test/tracing/protocols/graphql/mockVersion.js

```diff
@@ -1,61 +0,0 @@
-/*
```

Comment:
Q: I am wondering if we want to keep this file. It was really hard to mock a different graphql version.
     As soon as we add graphql v17 (which will come out soon), we need to the file again.

Reply:
Restored the file